Cabal/src/Distribution/Simple/Setup.hs

```diff
@@ -2262,11 +2262,11 @@ configureProg verbosity programDb prog = do
 -- | Helper function to split a string into a list of arguments.
 -- It's supposed to handle quoted things sensibly, eg:
 --
--- > splitArgs "--foo=\"C:/Program Files/Bar/" --baz"
--- >   = ["--foo=C:/Program Files/Bar", "--baz"]
+-- >>> splitArgs "--foo=\"C:/Program Files/Bar/" --baz"
```

Comment:
this should be `"--foo=\"C:/Program Files/Bar/\" --baz"` (backspace for the third double quote), I think (see also doctest CI failure)